packages/react-renderer-demo/src/components/component-example-text.js

```diff
@@ -7,12 +7,12 @@ import avalableMappers from '../helpers/available-mappers';
 const ComponentExampleText = ({ linkText, schema, variants, component, activeMapper, ContentText }) => (
   <React.Fragment>
     <Heading level="4" component="h1">
-      {linkText}
+      {`${avalableMappers.find(({ mapper }) => mapper === activeMapper).title} ${linkText}`}
```

Comment:
wouldn't this crash before the active mapper property is ready?

Reply:
MUI is always default.